### PR TITLE
Add build instructions to README

### PR DESCRIPTION
**Pull request by camisatx**
- Remove `Limitations` section from table of contents since it is not longer at the bottom
- Add table of contents section for `Build Instructions` and `ToDo`
- Add section with instructions for how to build the images

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -430,6 +431,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
 The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
 option.
 
+## Build Instructions
+
+Building your own images allows you to edit the configuration files of each image individually, providing more customization for your deployment.
```
Comment (saghul) [inline README.md]: Can you remove the extra space before the B?

File: README.md [hunk 2]
```diff
@@ -430,6 +431,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
...
+
+The docker images can be built by running the `make` command in the main repository folder. If you need to overwrite existing images from the remote source, use `FORCE_REBUILD=1 make`.
```
Comment (saghul) [inline README.md]: Ditto

File: README.md [hunk 3]
```diff
@@ -430,6 +431,16 @@ succeed. If your users are coming in over the Internet (and not over LAN), this
...
+
+If you are on the unstable branch, build the images with `FORCE_REBUILD=1 JITSI_RELEASE=unstable make`.
+
+Before deployment, make sure you remove the config file. You are now able to run `docker-compose up` as usual.
```
Comment (saghul) [inline README.md]: What config file? You should need to remove anything.

File: README.md (hunk 1 above)
Comment (camisatx) [reply README.md]: I think the space before `Build` is required for the md formatting, and I don't think there is a space before `Building`. The formatting in the diff may make it look like there is a space ;)

Comment (camisatx) [reply README.md]: Unless you are talking about the line 435? But that would conflict with the other titles in this doc.

File: README.md (hunk 3 above)
Comment (camisatx) [reply README.md]: Ah, that was in regard to any changes done in `.env`, which requires the removal of `.jitsi-meet-cfg`. But that doesn't matter if we are only building the docker images here, so out of scope for the build process itself. Agree?

Comment (saghul) [reply README.md]: Agreed!

Disregard the spacing thing, I see it ok now

**Comment by camisatx** [comment]
Okay I think we are good! Look good with you?

**Comment by saghul** [comment]
Cheers!

**Pull request merged**